assets/src/sass/_theme.scss

```diff
@@ -1,7 +1,5 @@
-.navbar-brand {
```

Comment:
I found a slight issue while testing this out. The top navbar is a slightly different size on the homepage vs. the code of conduct page after this change. `navbar-brand` has some built-in CSS associated with it from bootstrap and that's the reason why. Removing it makes the top navigation different sizes when navigating and that's much more awkward than an extra link.

![Screenshot from 2024-05-23 17-40-21](https://github.com/sandiegopython/pythonsd-django/assets/185043/44d4cf0f-e890-4a90-99fe-ea4c170687b2)
![Screenshot from 2024-05-23 17-40-32](https://github.com/sandiegopython/pythonsd-django/assets/185043/eb722cb0-6b55-4c1e-ac15-533a0171419b)

Reply:
This is odd. I'm seeing them exactly the same. I'm not sure where that 10px is coming from but it's not worth the trouble. I'll revert the CSS changes and go with the "#" solution.

![image](https://github.com/sandiegopython/pythonsd-django/assets/131234246/caf4a073-e9a6-4662-a611-9be7983127c4)

Reply:
You are right. It came from here: https://github.com/sandiegopython/pythonsd-django/pull/152#discussion_r1614130354